website_snippet_country_dropdown/views/snippets.xml

```diff
@@ -52,13 +51,14 @@
 
 <template id="country_dropdown" name="Country Dropdown">
     <t t-set="countries" t-value="request.env['res.country'].search([])"/>
-    <div id="snippet_container" class="input-group" t-ignore="true">
-        <div class="input-group-btn">
+    <t t-set="default_country" t-value="countries[:1]"/>
```

Comment:
The default values is set in website_sale_checkout_country_vat module, here only get the first country, in the future the user will can choose the default value for the snippet country in the snippet editor.